custom-scripts/rstudio_server.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This custom script only works on images where rstudio server is pre-installed on the Docker image
+# 
+# This custom script has been tested to work on the following docker images:
+#  - jiata/aztk-r:0.1.0-spark2.2.0-r3.4.1
+#  - jiata/aztk-r:0.1.0-spark2.1.0-r3.4.1
+#  - jiata/aztk-r:0.1.0-spark1.6.3-r3.4.1
+
+if  [ "$IS_MASTER" = "1" ]; then
+
+    ## Download and install Rstudio Server
```

Comment:
how long does this take? Can we package this into the docker image, and basically only do the setup here (instead of install too?).
Not sure if increase in time difference is worth the effort though.

Reply:
Downloading and installing Rstudio Server takes roughly 15 secs.. The installation package itself is only 60mb.

Also we only need rstudio server installed on the master. Not sure about terms of agreements are.. But we are only using Rstudio server open source. So I think it would be fine

Reply:
Rstudio server is [AGPLv3](https://support.rstudio.com/hc/en-us/articles/217801078-What-license-is-RStudio-available-under-). So unmodified redistribution, I believe, is fine.

This is the relevant section of the license:
`To "modify" a work means to copy from or adapt all or part of the work in a fashion requiring copyright permission, other than the making of an exact copy. The resulting work is called a "modified version" of the earlier work or a work "based on" the earlier work.`